Classes/LinphoneCoreSettingsStore.m

```diff
@@ -283,7 +283,8 @@ - (void)transformLinphoneCoreToKeys {
         }
 		[self setCString:preset ? preset : "high-fps" forKey:@"video_preset_preference"];
         MSVideoSize vsize;
-        
+        vsize.width = 0;
+        vsize.height = 0;
```

Comment:
Just the initial initialisation of variable

Xcode 7.3 gives error without this initialization. 
"Variable 'vsize' is used uninitialized whenever 'if' condition is false"